rust/tvm/src/ir/module.rs

```diff
@@ -156,4 +145,127 @@ impl IRModule {
     {
         module_lookup_str(self.clone(), name.into())
     }
+
+    pub fn get_global_type_vars(&self) -> Result<Array<GlobalTypeVar>> {
+        module_get_global_type_vars(self.clone())
```

Comment:
Is `clone()` on an `IRModule` expensive?

Reply:
No all the "language" types are a newtype around an Option<ObjectPtr<T>> and ObjectPtr<T> is roughly Arc<T> plus inheritance support.